jwql/shared_tasks/shared_tasks.py

```diff
@@ -549,7 +549,7 @@ def prep_file(input_file, in_ext):
         logging.critical(msg.format(short_name))
         raise ValueError("Redis lock for {} is in an unknown state".format(short_name))
     logging.info("\t\tAcquired Lock.")
-    logging.info("\t\tCopying {} to {}".format(input_file, send_path))
+    logging.info("\t\tCopying {} to {}".format(uncal_file, send_path))
```

Comment:
Okay, I see the change @bhilbert4 I will go ahead and merge develop and then add this in!